signer/src/storage/postgres.rs

```diff
@@ -2121,6 +2152,50 @@ impl super::DbRead for PgStore {
         .map_err(Error::SqlxQuery)
     }
 
+    async fn is_withdrawal_active(
+        &self,
+        id: &model::QualifiedRequestId,
+        bitcoin_chain_tip: &model::BitcoinBlockRef,
+        min_confirmations: u64,
+    ) -> Result<bool, Error> {
+        let min_block_height = sqlx::query_scalar::<_, Option<i64>>(
```

Comment:
Comment suggestion:

```suggestion
        // We look for all `bitcoin_withdrawal_outputs` for the given request id for which
        // we have a matching bitcoin block and return the highest block height we find (if any).
        let min_block_height = sqlx::query_scalar::<_, Option<i64>>(
```

Reply:
Done in https://github.com/stacks-network/sbtc/pull/1411/commits/3b8bfbaaee84886e4a2433d635a55b6b8714f2ad